lib/iris/cube.py

```diff
@@ -2500,6 +2544,24 @@ def vector_summary(
                 summary += "\n     Ancillary variables:\n"
                 summary += "\n".join(ancillary_variable_summary)
 
+            #
+            # Generate summary of ugrid mesh object.
+            #
+            if ugrid_mesh:
+                ugrid_mesh_summary, cube_header = vector_summary(
+                    [], cube_header, max_line_offset, ugrid_mesh=ugrid_mesh,
```

Comment:
We're using black to control the formatting of our code. Black didn't use line breaks between arguments because it all fits on one line, which isn't the case for the other ones